assets/dev/licenses.json

```diff
@@ -0,0 +1,6826 @@
+{
```

Comment:
Just wondering - does this file exist as some kind of a package that we can pull from the composer? Because maintaining this could be an issue.

Reply:
It doesn't, but I wanted to hear your feedback before I went too far lol.  It's a mix from a couple of sources, so I was thinking a build script to fetch and arrange the data would be good for updates.  It could possibly go in the plugin instead to retrieve the remote sources with a long transient since they aren't usually updated often.  It's from the spdx licenses json repo and a fsf scrape ( the fsf scrape isn't entirely accurate so i had to change a few things too ).

Reply:
Maybe you could publish it in the `npm/packagist`, that way the maintenance of that file would be separate from the Theme Sniffer, and we could pull it in vendors if something changes (just a version bump). What do you think?

Reply:
yeah i'll try to get to that this week - sounds like a good way to go :+1: